libbeat/publisher/pipeline/pipeline_test.go

```diff
@@ -79,7 +79,7 @@ func TestPipelineAcceptsAnyNumberOfClients(t *testing.T) {
 // close method is called, this ID is returned
 func makeDiscardQueue() queue.Queue {
 	var wg sync.WaitGroup
-	producerID := atomic.NewInt(0)
+	producerID := atomic.Int64{}
```

Comment:
```suggestion
	var producerID atomic.Int64
```
but merge into single var block with `wg` above.